### DISCUSSION TITLE
Why is Option API not recommanded in NuxtJS 3?

### DISCUSSION THREAD

**Discussion by abeldotam** [nuxt/nuxt#18251] [Questions] [answered]
Hi folks 👋 

From what I can see, the option API isn't recommended in NuxtJS 3 ([see the comparaison here](https://v3.nuxtjs.org/getting-started/introduction#comparison)) and I would like to understand the meaning.

![image](https://user-images.githubusercontent.com/5216201/138941677-3a306806-eef1-46c2-be1d-8e6224587ce5.png)

There is a lot of discussions(*) about the pros and cons of Composition API on Reddit, Twitter, etc...
Some love it, others doesn't, the conclusions are often the same: it's a tool and a choice like any other, to be used where and when appropriate.

**As far I know** (and please let me kindly know if I'm wrong 😄 ), Vue.js's core team doesn't officially position itself more than that on the future of Option API vs Composition API, the two must and will coexist together, so I wonder about the position of NuxtJS on that side.

Is it simply that it isn't **currently** recommended to use the option API for technical questions (ex: not yet well implemented) or is it a real directive from the NuxtJS team to want to abandon the API option (which would be problematic for developers not intending to mess with API composition)?

As I wanted to start a pet project for testing NuxtJS 3, I'm just trying to understand. 🙂 
Take care.

_(* : ... And this post is not intended to become yet another debate on this subject)_

**Comment by danielroe** [comment, marked as answer]
See https://github.com/nuxt/framework/discussions/1075#discussioncomment-1483487

**Reply by abeldotam** [reply]
Thanks!
